### PR TITLE
Make the Python pitch more concise

### PR DESCRIPTION
**Pull request by evandandrea**
This updates the copy for Python in the first snap flow to be more concise, a collaboration between @therealjuan and myself.

### CODE REVIEW

File: templates/home/_fsf_python.html [hunk 1]
```diff
@@ -2,16 +2,21 @@
...
     <h4>Why are snaps good for Python projects?</h4>
     <ul>
-      <li>Snaps are easy to discover and install. Millions of users can browse and install snaps graphically in the Snap Store or from the command-line.</li>
-      <li>Snaps install and run the same across Linux. They bundle the exact version of Python required and all of your app’s dependencies, be they Python modules or system libraries.</li>
-      <li>Snaps automatically update to the latest version. Four times a day, users’ systems will check for new versions and upgrade in the background.</li>
-      <li>Upgrades are not disruptive. Because upgrades are not in-place, users can keep your app open as it’s upgraded in the background.</li>
-      <li>Upgrades are safe. If your app fails to upgrade, users automatically roll back to the previous revision.</li>
+      <li>Easy to discover and install by millions using the Snap Store or command-line every day</li>
+      <li>Automatically updated to the latest stable version of your app</li>
+      <li>Revert to previous version if an update fails, preserving data</li>
```
Comment (Lukewh) [inline templates/home/_fsf_python.html]: Reads like it's missing a word 'Revert to the previous...' or 'Revert to a previous...' or something completely different?

Comment (evandandrea) [reply templates/home/_fsf_python.html]: Fixed in 90b34e6.

**Comment by therealjuan** [comment]
Might be a good idea to do a separate PR updating the content on Go but apart from that LGTM 👍

**Comment by evandandrea** [comment]
100% the plan :)

**Pull request merged**